src/md/tekwill/jf4/homework/Exercise4.java

```diff
@@ -18,5 +18,14 @@
 
     public static void main(String[] args) {
 
+        int number1, number2, number3, number4;
+
+        number1 = -5 + 8 * 6;
+        number2 = (55 + 9) % 9;
+        number3 = 20 + -3*5 / 8;
+        number4 = 5 + 15 / 3 * 2 - 8 % 3;
+
+        System.out.println("Rezultatul operatiilor este:\n" +number1+"\n"+number2+"\n"+number3+"\n"+number4);
```

Comment:
Good work? but a, b, c, d should be tested/ passed to your program separatelly, prefferable via argument parameter